sensor_msgs/PointCloud2 pcl
string label
